### ISSUE TITLE
Staging Review finding: Form error messages not announced to screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99401]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medical Records [Download Records](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92340), [Flow/Images](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95081) and [Settings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95082)

## Product Information
**Team:** vfs-mhv-medical-records
**Product:** MHV Medical Records
**Feature:** Phase 1 Radiology Flow/Images, Download Records, and Settings

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the Blue Button form flow, fields at each step of the flow are required. If you press Continue without proviidng a response, an error message is triggered on those fields.

However, those error messages are not announced to screen reader users. Instead, when pressing Continue, you don't hear anything. Silence can be disorienting and confusing when you don't have any visual cues to explain what's happening. Did the button not work? Should you try pressing it again? Is this tool just broken? Or did you make a mistake somewhere? The user has to guess what happened, which is a difficult position to put them in.

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/71539ee2-3da2-41c5-a033-0860506e2f2b

</details>

## Recommended action

I had been under the impression that error validation comes baked in with the forms library, but I'll admit I don't have a deep understanding of how the forms library works. I recommend matching the error handling behavior in the [mock form patterns](https://staging.va.gov/mock-form-patterns/introduction) and other production VA.gov forms, and to reach out to the forms team as needed for implementation guidance.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by acrollet** [comment]
See the recommended action and linked PR [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/89010) for a possible workaround. @shiragoodman @humancompanion-usds should we explore whether this can or should be fixed in the DS components? Maybe a judicious application of `aria-live` could resolve this without consumers having to manipulate the focus?

cc @kfink24

**Comment by shiragoodman** [comment]
@briandeconinck FYI

**Comment by bryan-riley-va** [comment]
Resolved, tested, merged as part of https://jira.devops.va.gov/browse/MHV-65352
